build.gradle

```diff
@@ -22,85 +21,131 @@ group 'io.xh'
 apply plugin:'idea'
 apply plugin:'war'
 apply plugin:'org.grails.grails-web'
-apply plugin:'org.grails.grails-gsp'
 
 repositories {
     mavenLocal()
-    mavenCentral()
```

Comment:
Why are we removing mavenCentral here and above?  Do we want to ensure we're getting everything from the Grails repo (assuming it has a mirror of all the deps we need)?

For the record, I would not want to lean any more on our XH repo as a pass-through source for dependencies - hopefully we can do away with that soon altogether.